src/legacy/core_plugins/metrics/public/components/_vis_with_splits.scss

```diff
@@ -1,11 +1,17 @@
 .tvbSplitVis {
   display: flex;
   flex: 1 0 auto;
-}
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  width: 100%;
 
-.tvbSplitVis__split {
-  display: flex;
-  flex: 1 0 0;
-  flex-direction: column;
-  overflow: hidden;
+  &__split {
```

Comment:
We've been trying to avoid this style of concatenating class names because it makes it harder to debug when searching for the entire class name.